frontend/src/routes.tsx

```diff
@@ -17,6 +18,7 @@ export const routes = [
   { path: "/startups/:id", element: <StartupView /> },
   { path: "/dashboard", element: <InvestorDashboard /> },
   { path: "/messages", element: <Inbox /> },
-  { path: "/forgot-password", element: <PasswordResetRequest /> },
+  { path: "/reset-password", element: <PasswordResetRequest /> },
```

Comment:
Changing the public route path from /forgot-password to /reset-password is an externally visible API change. If backend-generated reset emails or other places still link to /forgot-password, users will receive broken links.
Verify and update all places that generate or reference the reset URL (backend email templates, environment config, documentation). Consider adding a redirect from the old path to the new one to maintain backward compatibility